scanner/cmd/scannerctl/scale.go

```diff
@@ -73,13 +122,42 @@ func scaleCmd(ctx context.Context) *cobra.Command {
 		"index-only",
 		false,
 		"Only index the specified image")
+
+	indexerCacheState := flags.String("indexer-state", "cold", "Was the index caching warm or cold.")
+
+	pprofDir := flags.String(
+		"pprof-dir",
+		"pprof",
+		"Directory to save pprof data captured from the indexer and matcher")
+
+	pprofEnabled := flags.Bool(
+		"pprof",
+		false,
+		"Profile both Indexer and Matcher using pprof endpoints at port ")
+
 	cmd.RunE = func(cmd *cobra.Command, args []string) error {
+		flags.VisitAll(func(f *pflag.Flag) {
+			log.Printf("%s=%s", f.Name, f.Value.String())
+		})
+
+		go func() {
+			http.Handle("/metrics", promhttp.Handler())
```

Comment:
do we always want to start this or just when pprof is enabled?

Reply:
This is unrelated to pprof.

Nonetheless, it's a good idea to be able to run this without opening a port on your host.

I rename `--pprof` to reflect both use cases.